action/protocol/execution/evm/evm.go

```diff
@@ -421,74 +421,65 @@ func executeInEVM(ctx context.Context, evmParams *Params, stateDB *StateDBAdapte
 }
 
 // evmErrToErrStatusCode returns ReceiptStatuscode which describes error type
-func evmErrToErrStatusCode(evmErr error, g genesis.Blockchain, height uint64) (errStatusCode uint64) {
+func evmErrToErrStatusCode(evmErr error, g genesis.Blockchain, height uint64) uint64 {
+	errStatusCode := iotextypes.ReceiptStatus_Failure
+	if !g.IsBering(height) {
```

Comment:
Why is this add?

Reply:
what is simplified?

Reply:
see below removed block of code in `switch`